src/app.jsx

```diff
@@ -31,52 +31,47 @@ const App = () => {
       )}
 
       <div className="actions">
-        <button
-          type="button"
-          onClick={() => setPrintModal(!printModal)}
-        >
-          {messages.add}
-        </button>
+        <div className="actions__secondary">
+          <button
+            type="button"
+            onClick={() => setPrintModal(!printModal)}
+          >
+            {messages.add}
+          </button>
 
-        <button
-          type="button"
-          onClick={clear}
-        >
-          {messages.clear}
-        </button>
+          <button
+            type="button"
+            onClick={clear}
+          >
+            {messages.clear}
+          </button>
+        </div>
 
         <button
           type="button"
           onClick={() => dispatch('@@ui/ROLL>DICES')}
+          className="actions__primary"
         >
-          {messages.rollAllDices}
+          {dices.length > 1 ? messages.rollAllDices : messages.roll}
         </button>
       </div>
 
-      <div className="stats total">
-        {dices.length > 1 && (
-          <Fragment>
-            <span className="label">
-              {messages.total}
-            </span>
-            <span>
-              {dices.reduce((acc, { roll = 0 }) => acc + roll, 0)}
-            </span>
-          </Fragment>
-        )}
+      <div className={`stats substract ${dices.length !== 2 && 'inactive'}`}>
```

Comment:
We can use classnames if you prefer it https://bundlephobia.com/result?p=classnames@2.2.6